features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/vitals/JankStatsActivityLifecycleListenerTest.kt

```diff
@@ -356,10 +359,41 @@ internal class JankStatsActivityLifecycleListenerTest {
         verify(mockObserver, never()).onNewSample(any())
     }
 
+    @Test
+    fun `𝕄 adjust sample value to refresh rate 𝕎 doFrame() {refresh rate over 60hz}`(
+        @LongForgery timestampNs: Long,
+        @LongForgery(ONE_MILLISECOND_NS, ONE_SECOND_NS) frameDurationNs: Long,
+        @BoolForgery isJank: Boolean,
+        @DoubleForgery(60.0, 120.0) displayRefreshRate: Double
+    ) {
+        // Given
+        val expectedFrameRate = ONE_SECOND_NS.toDouble() / frameDurationNs.toDouble()
+        val refreshRateMultiplier = 60.0 / displayRefreshRate
+        val frameData = FrameData(timestampNs, frameDurationNs, isJank, emptyList())
+
+        val variableRefreshRateListener = JankStatsActivityLifecycleListener(
+            mockObserver,
+            mockInternalLogger,
+            mockJankStatsProvider,
+            displayRefreshRate
+        )
+        // When
+        variableRefreshRateListener.onFrame(frameData)
+
+        // Then
+        if (expectedFrameRate * refreshRateMultiplier > MIN_FPS) {
+            verify(mockObserver).onNewSample(eq(min(expectedFrameRate * refreshRateMultiplier, MAX_FPS), 0.0001))
+        } else {
+            verify(mockObserver, never()).onNewSample(any())
+        }
+    }
+
     companion object {
         const val ONE_MILLISECOND_NS: Long = 1000L * 1000L
         const val ONE_SECOND_NS: Long = 1000L * 1000L * 1000L
         const val TEN_SECOND_NS: Long = 10L * ONE_SECOND_NS
         const val ONE_MINUTE_NS: Long = 60L * ONE_SECOND_NS
+        const val MIN_FPS: Double = 1.0
```

Comment:
minor: If we in the future modify min/max fps we would have to change this in two places. We could make the class variables internal to share with the test

Reply:
I think it's unlikely to change, but I can follow this pattern if we're using it elsewhere.